grid-client/calculator/calculate.go

```diff
@@ -44,9 +45,10 @@ func (c *Calculator) CalculateCost(cru, mru, hru, sru int64, publicIP, certified
 	if certified {
 		certifiedFactor = 1.25
 	}
-
+	// cost per month in mUSD
 	costPerMonth := (cu*float64(pricingPolicy.CU.Value) + su*float64(pricingPolicy.SU.Value) + ipv4*float64(pricingPolicy.IPU.Value)) * certifiedFactor * 24 * 30
-	return costPerMonth / float64(tftPrice) / 1000, nil
+	// convert to USD
+	return costPerMonth / mUSDToUSD, nil
```

Comment:
I don't get why we removed the tfpirce from the equation

Reply:
the cost already in mUSD, so we only need to convert it to USD